src/main/webapp/js/instructorFeedbackResultsAjaxByGQR.js

```diff
@@ -67,7 +67,9 @@ $(document).ready(function() {
                 $('a[id^="collapse-panels-button-section-"],a[id^="collapse-panels-button-team-"]').off('click');
                 $('a[id^="collapse-panels-button-section-"]').on('click', function() {
                     var isGroupByTeam = $('#frgroupbyteam').prop('checked');
-                    var childPanelType = isGroupByTeam ? 'div.panel.panel-warning' : 'div.panel.panel-primary';
```

Comment:
Instead of duplicating the fix to 4 different files, can you explore using delegated events in `instructorFeedbackResults.js`? Most of the duplicate code regarding this section button can be removed in this PR.